### PR TITLE
Fix typing errors

### PR DESCRIPTION
**Pull request by dforsi**
Found using codespell.

### CODE REVIEW

**Comment by dforsi** [comment]
Some changes in this PR are in printf strings, but most changes are in comments.
Let me know if you prefer to have two separate commits.
I tried to avoid changing third party code, like snprintf.c and I will amend this PR if you think that some changes should be reverted.

thanks,
Daniele

**Comment by tvrusso** [comment]
I have reviewed these changes and most appear to be legitimate spelling errors.  The only exception I have found is in License where "du Pont" has been changed incorrectly to "du Point".

If you amend that I will review it again.

File: LICENSE [hunk 1]
```diff
@@ -347,9 +347,9 @@ The licences which components of this software fall under are as follows.
 
 3)
   From 1991 to August 1999, ImageMagick was developed and distributed by
-  E. I. du Pont de Nemours and Company:
+  E. I. du Point de Nemours and Company:
 
-  Copyright 1999 E. I. du Pont de Nemours and Company
+  Copyright 1999 E. I. du Point de Nemours and Company
```
Comment (tvrusso) [inline LICENSE:352]: The company name is "du Pont" not "du Point" and so it is incorrect to "fix" this "error."

Comment (dforsi) [reply LICENSE]: Indeed, fixed now.

**Pull request merged**